x-pack/packages/ml/aiops_components/src/dual_brush/dual_brush.tsx

```diff
@@ -169,8 +169,9 @@ export function DualBrush({
           };
 
           if (
-            baselineSelection[0] !== newBrushPx.baselineMin ||
-            baselineSelection[1] !== newBrushPx.baselineMax
+            id === 'baseline' &&
```

Comment:
Small suggestion to have 'baseline' and 'deviation' as constants, which can be used in `x-pack/packages/ml/aiops_components/src/dual_brush/dual_brush.tsx` as well